Palindrome/palindrome_string.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+#include <string.h> /*used string library header file for string functions.*/
+main()
+{
+	char str[200], rev[200];
+	printf ("Enter a string : ");
+	gets (str); /*This function is used to read multiword string (one line) from keyboard*/
+	strcpy (rev,str); /*This function is used to copy one string into another*/
```

Comment:
Add space after comma and around operators

Reply:
I have added space after comma operators.
Can you please tell what are these around operators.